### PR TITLE
Class features

### PR DESCRIPTION
**Pull request by aptuckey**
I fixed the bug that occurred when running delete or rename class. 
I created a temp list an stored the tuple keys there, then deleted them from the dictionary

### CODE REVIEW

**Comment by aptuckey** [comment]
Sorry for the notification spam. This is ready for review

File: ClassCollection.py [hunk 1]
```diff
@@ -56,24 +61,28 @@ def deleteClass(self, name):
...
-                        self.deleteRelationship(oldName, name1)
+                    toChange.append(theTuple)            
+                               
+            for theTuple in toChange:
+                (name1, name2) = theTuple
+                if name1 == oldName:
+                    self.addRelationship(newName,name2)
+                    self.deleteRelationship(oldName, name2)
+                else:
+                    self.addRelationship(newName,name1)
```
Comment (sfmalloy) [inline ClassCollection.py:83]: should be `self.addRelationship(name1, newName)`

File: ClassCollection.py [hunk 2]
```diff
@@ -56,24 +61,28 @@ def deleteClass(self, name):
...
+                    self.deleteRelationship(oldName, name1)
```
Comment (sfmalloy) [inline ClassCollection.py:84]: should be `self.addRelationship(name1, newName`

**Review by sfmalloy — CHANGES_REQUESTED** [review]

File: ClassCollection.py (hunk 1 above)
Comment (sfmalloy) [inline ClassCollection.py]: line 83: should be `self.addRelationship(name1, newName)`

File: ClassCollection.py (hunk 2 above)
Comment (sfmalloy) [inline ClassCollection.py]: line 84: should be `self.deleteRelationship(name1, oldName)`

**Review by sfmalloy — CHANGES_REQUESTED** [review]

**Review by sfmalloy — APPROVED** [review]
Looks good!

**Pull request merged**